### DISCUSSION TITLE
Stop a running job

### DISCUSSION THREAD

**Discussion by IvanovskyOrtega** [dkron-io/dkron#1261] [Q&A]
Hi!
I've been using dkron for a while and everything runs flawlessly, except for something that maybe I'm missing.

Is there a way to stop a running job?

I'm running dkron in kubernetes and, by now, the only way I'd thought to stop a job in execution is deleteting the agent's pod running that job.  I'd like to know if there's something else to stop'em.

**Comment by yvanoers** [comment]
There's no explicit kill command in Dkron (that I'm aware of).
The shell executor does have a timeout available that (attempts to) kill the proces if it runs longer than the timeout.

A work around would be to log into the pod that's running the job you want to kill and kill it in there. I realize that this is not a very friendly approach, but if available to you, it less blunt than deleting an entire pod.
